src/components/Box/Box.types.ts

```diff
@@ -0,0 +1,338 @@
+import { SpacingKey } from 'theme/globals/spacing';
+import { BackgroundColorKey } from 'theme/tokens/semantic/backgroundColor';
+import { TextColorKey } from 'theme/tokens/semantic/textColor';
+
+export type Responsive<T> = T | Array<T>;
+
+type AlignContent =
+  | 'center'
+  | 'start'
+  | 'end'
+  | 'flex-start'
+  | 'flex-end'
+  | 'normal'
+  | 'baseline'
+  | 'first baseline'
+  | 'last baseline'
+  | 'space-between'
+  | 'space-around'
+  | 'space-evenly'
+  | 'stretch'
+  | 'safe center'
+  | 'unsafe center'
+  | 'inherit'
+  | 'initial'
+  | 'unset';
+type AlignItems =
```

Comment:
[optional] is there a way maybe to group types which equal the same unions?

e.g.:
type AlignBase = 'normal' | 'stretch| ........

type AlignItems = AlignBase
type AlignSelf = AlignBase

thoughts?

Reply:
are not 100% the same and even if they are is a really small win imo